WooCommerce/Classes/Authentication/Store Creation/StoreCreationCoordinator.swift

```diff
@@ -7,6 +7,10 @@ import protocol Storage.StorageManagerType
 
 /// Coordinates navigation for store creation flow, with the assumption that the app is already authenticated with a WPCOM user.
 final class StoreCreationCoordinator: Coordinator {
+    enum LocalNotificationUserInfoKey {
```

Comment:
nit: this doesn't feel like belonging to `StoreCreationCoordinator` 🤔 what about in `LocalNotification` or some other struct/enum that's more directly related?

Reply:
I agree. Moved to `LocalNotification`in https://github.com/woocommerce/woocommerce-ios/pull/9749/commits/1614fc41fa7b047e5b7e11b8ef9a8da6e9c2b645